packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 48.9.0 - April 2, 2020
+
+## Fixed
+
+#### docs
+
+- Fixed documentation for [Prettier Setup Requirements](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/prettier) ([PR #2105](https://github.com/forcedotcom/salesforcedx-vscode/pull/2105), [Issue #2079](https://github.com/forcedotcom/salesforcedx-vscode/issues/2079))
+
+## Added
```

Comment:
I left out W-6531294, W-6531281, W-6531288 for now since they are still in a QA state. We can always release them whenever ready like we discussed on Tuesday. Let me know if you think it'd be good to include some notes on the change for this release.

Reply:
@sfsholden no need for adding those here since those changes are part of the new library.